pkg/apis/pravega/v1beta1/pravega.go

```diff
@@ -167,6 +168,9 @@ type PravegaSpec struct {
 
 	// SegmentStoreExternalTrafficPolicy defines the ExternalTrafficPolicy it can have cluster or local
 	SegmentStoreExternalTrafficPolicy string `json:"segmentStoreExternalTrafficPolicy,omitempty"`
+
+	// SecurityContext holds security configuration that will be applied to a container
+	SecurityContext *corev1.PodSecurityContext `json:"securityContext,omitempty"`
```

Comment:
The PodSecurityContext is getting applied to the segment store only in this PR.     And indeed the controller pods don't necessarily need the same PSC.    (in particular, SegmentStores pod have more requirements in terms of requiring access to NFS for example, versus controllers which require less).

While the changes look correct, perhaps calling out that this is only getting applied to the SS, (renaming it accordingly?) , or also adding a separate one for the controller pods as well, might help avoid confusion?

Reply:
I have changed the name and added securitycontext for controller as well.